transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/LogMessages.kt

```diff
@@ -1,3 +1,97 @@
 package com.genesys.cloud.messenger.transport.util.logs
 
-object LogMessages
+object LogMessages {
```

Comment:
Just an overall organization comment. 

It might be helpful if we organize logs by theme. like attachment related logs in one section, message logs in other, general logs, etc.

You can use comment for that. Something like 
// Attachment
attachment logs goes here...